Portal/View/Components/MapView.swift

```diff
@@ -32,7 +32,7 @@ public struct MapPlacemark {
     
 }
 
-public struct MapProperties {
+public struct MapProperties: AutoPropertyDiffable {
     
     public var placemarks: [MapPlacemark]
```

Comment:
How is the comparison being made in the autogenerated code by `AutoPropertyDiffable` for this property? Is it comparing both arrays using the `==` operator?

Reply:
``` swift code 
public func changeSet(for mapProperties: MapProperties) -> [MapProperties.Property] {
        var changeSet: [MapProperties.Property] = []
        if self.placemarks != mapProperties.placemarks {
            changeSet.append(.placemarks(mapProperties.placemarks))
        }

        ...
}
```